stable/redmine/templates/cronjob.yaml

```diff
@@ -0,0 +1,140 @@
+            - name: REDMINE_MAIL_RECEIVER_PORT
+              value: {{ .Values.mailReceiver.port | quote }}
+            - name: REDMINE_MAIL_RECEIVER_USERNAME
+              value: {{ .Values.mailReceiver.username | quote }}
+            - name: REDMINE_MAIL_RECEIVER_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "redmine.secretName" . }}
+                  key: mail-receiver-password
+            - name: REDMINE_MAIL_RECEIVER_USE_SSL
+              value: {{ .Values.mailReceiver.ssl | quote }}
+            - name: REDMINE_MAIL_RECEIVER_STARTTLS
+              value: {{ .Values.mailReceiver.startTLS | quote }}
+            - name: REDMINE_MAIL_RECEIVER_IMAP_FOLDER
+              value: {{ .Values.mailReceiver.imapFolder | quote }}
+            - name: REDMINE_MAIL_RECEIVER_IMAP_MOVE_ON_SUCCESS
+              value: {{ .Values.mailReceiver.moveOnSuccess | quote }}
+            - name: REDMINE_MAIL_RECEIVER_IMAP_MOVE_ON_FAILURE
+              value: {{ .Values.mailReceiver.moveOnFailure | quote }}
+            - name: REDMINE_MAIL_RECEIVER_UNKNOWN_USER_ACTION
+              value: {{ .Values.mailReceiver.unknownUserAction | quote }}
+            - name: REDMINE_MAIL_RECEIVER_NO_PERMISSION_CHECK
+              value: {{ .Values.mailReceiver.noPermissionCheck | quote }}
+            - name: REDMINE_MAIL_RECEIVER_NO_ACCOUNT_NOTICE
+              value: {{ .Values.mailReceiver.noAccountNotice | quote }}
+            - name: REDMINE_MAIL_RECEIVER_DEFAULT_GROUP
+              value: {{ .Values.mailReceiver.defaultGroup | quote }}
+            - name: REDMINE_MAIL_RECEIVER_PROJECT
+              value: {{ .Values.mailReceiver.project | quote }}
+            - name: REDMINE_MAIL_RECEIVER_PROJECT_FROM_SUBADDRESS
+              value: {{ .Values.mailReceiver.projectFromSubaddress | quote }}
+            - name: REDMINE_MAIL_RECEIVER_STATUS
+              value: {{ .Values.mailReceiver.status | quote }}
+            - name: REDMINE_MAIL_RECEIVER_TRACKER
+              value: {{ .Values.mailReceiver.tracker | quote }}
+            - name: REDMINE_MAIL_RECEIVER_CATEGORY
+              value: {{ .Values.mailReceiver.category | quote }}
+            - name: REDMINE_MAIL_RECEIVER_PRIORITY
+              value: {{ .Values.mailReceiver.priority | quote }}
+            - name: REDMINE_MAIL_RECEIVER_ASSIGNED_TO
+              value: {{ .Values.mailReceiver.assignedTo | quote }}
+            - name: REDMINE_MAIL_RECEIVER_ALLOW_OVERRIDE
+              value: {{ .Values.mailReceiver.allowOverride | quote }}
+            - name: REDMINE_MAIL_RECEIVER_SECRET_KEY_BASE
+              value: {{ randAlphaNum 64 | quote }}
+            - name: REDMINE_MAIL_RECEIVER_PROTOCOL
+              value: {{ .Values.mailReceiver.mailProtocol | quote }}
+            - name: REDMINE_MAIL_RECEIVER_POP3_DELETE_UNPROCESSED
+              value: {{ .Values.mailReceiver.moveOnFailure | quote }}
```

Comment:
Could you please indent arrays with two spaces? Just for consistency with the rest of the chart
Use:
```yaml
array:
  - aaa: xxx
  - bbb: yyy
```
Instead of:
```yaml
array:
- aaa: xxx
- bbb: yyy
```

Reply:
Thank you. Fixed.